lib/samples/create_buffers_around_points/create_buffers_around_points.dart

```diff
@@ -0,0 +1,410 @@
+
+  // Fill symbols for buffer and tap points.
+  final bufferFillSymbol = SimpleFillSymbol();
+  final tapPointSymbol = SimpleMarkerSymbol();
+
+  final statePlanNorthCentralTexasSpatialReference =
+      SpatialReference(wkid: 32038);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: true,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                    onTap: onTap,
+                  ),
+                ),
+                Row(
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    ElevatedButton(
+                      onPressed: () {
+                        setState(() => _showSettings = !_showSettings);
+                      },
+                      child: const Text('Settings'),
+                    ),
+                    // A button to clear the buffers.
+                    ElevatedButton(
+                      onPressed: _bufferPoints.isEmpty
+                          ? null
+                          : () {
+                              clearBufferPoints();
+                              setState(() => _status = Status.addPoints);
+                            },
+                      child: const Text('Clear'),
+                    ),
+                  ],
+                ),
+                // Conditionally display the settings.
+              ],
+            ),
+            // Display a progress indicator and prevent interaction until state is ready.
+            Visibility(
+              visible: !_ready,
+              child: const SizedBox.expand(
+                child: ColoredBox(
+                  color: Colors.white30,
+                  child: Center(child: CircularProgressIndicator()),
+                ),
+              ),
+            ),
+            // Display a banner with instructions at the top.
+            SafeArea(
+              child: IgnorePointer(
+                child: Container(
+                  padding: const EdgeInsets.all(10),
+                  color: Colors.white.withOpacity(0.7),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      Text(
+                        _status.label,
+                        textAlign: TextAlign.center,
+                        style: const TextStyle(color: Colors.black),
+                      ),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+      bottomSheet: _showSettings? buildSettings(context, setState) : null,
```

Comment:
I feel like `dart format` would've added a space before the ? in `_showSettings?`. If you're running Android Studio, is there a way to configure `dart format` to run whenever the file is saved? There's a way to do that in VSCode.

Reply:
Yes, I'll set it up to run from now on.